examples.py/3D/Camera/OrthoVSPerspective.py

```diff
@@ -1,36 +1,41 @@
 """
- * Ortho vs Perspective.
- *
- * Click to see the difference between orthographic projection
- * and perspective projection as applied to a simple box.
- * The ortho() function sets an orthographic projection and
- * defines a parallel clipping volume. All objects with the
- * same dimension appear the same size, regardless of whether
- * they are near or far from the camera. The parameters to this
- * function specify the clipping volume where left and right
- * are the minimum and maximum x values, top and bottom are the
- * minimum and maximum y values, and near and far are the minimum
- * and maximum z values.
+Ortho vs Perspective.
+
+Click to see the difference between orthographic projection
+and perspective projection as applied to a simple box.
+The ortho() function sets an orthographic projection and
+defines a parallel clipping volume. All objects with the
+same dimension appear to be the same size, regardless of whether
+they are near or far away from the camera. The parameters to this
+function specify the clipping volume where left and right
+are the minimum and maximum x values respectively, 
+top and bottom are the minimum and maximum y values respectively, 
+and lastly, the near and far are the minimum and maximum z values.
  """
+showPerspective = True
 def setup():
     size(640, 360, P3D)
-    noStroke()
+    noFill()
     fill(204)
+    noStroke()
+ 
 
 def draw():
-    background(0)
     lights()
-
-    if mousePressed:
-        fov = PI / 3.0
-        cameraZ = (height / 2.0) / tan(PI * fov / 360.0)
-        perspective(fov, float(width) / float(height),
-                                cameraZ / 2.0, cameraZ * 2.0)
+    background(0)
+    far = map(mouseX, 0, width, 120, 400)
+    if showPerspective:
+        perspective(PI / 3.0, float(width) / float(height), 10, far)
```

Comment:
Those **float()** conversions are redundant b/c upcasting from integral to fractional type is automatic.